CI failing on main

We have failing CI on main...I am guessing this is an updated S2 of some kind (perhaps s2coding moved).

```
[ 12%] Building CXX object CMakeFiles/s2geography.dir/src/s2geography/accessors-geog.cc.o
In file included from /Users/runner/work/s2geography/s2geography/src/s2geography/accessors-geog.cc:2:
In file included from /Users/runner/work/s2geography/s2geography/src/s2geography/accessors-geog.h:6:
In file included from /Users/runner/work/s2geography/s2geography/src/s2geography/aggregator.h:4:
/Users/runner/work/s2geography/s2geography/src/s2geography/geography.h:312:24: error: use of undeclared identifier 's2coding'
  312 |   void set_coding_hint(s2coding::CodingHint hint) { hint_ = hint; }
      |                        ^
/Users/runner/work/s2geography/s2geography/src/s2geography/geography.h:313:3: error: use of undeclared identifier 's2coding'
  313 |   s2coding::CodingHint coding_hint() const { return hint_; }
      |   ^
/Users/runner/work/s2geography/s2geography/src/s2geography/geography.h:333:3: error: use of undeclared identifier 's2coding'
  333 |   s2coding::CodingHint hint_{s2coding::CodingHint::FAST};
      |   ^
3 errors generated.
make[2]: *** [CMakeFiles/s2geography.dir/src/s2geography/accessors-geog.cc.o] Error 1
make[1]: *** [CMakeFiles/s2geography.dir/all] Error 2
make: *** [all] Error 2

```

--- a/.github/workflows/run-tests.yaml
+++ b/.github/workflows/run-tests.yaml
@@ -80,7 +80,6 @@ jobs:
         shell: bash -l {0}
         run: |
           cmake --build build --config Release
-
       - name: Run tests
         shell: bash -l {0}
         run: |

--- a/ci/environment.yml
+++ b/ci/environment.yml
@@ -3,6 +3,6 @@ channels:
   - conda-forge
 dependencies:
   - cxx-compiler
-  - s2geometry
+  - s2geometry >=0.11
   - libabseil
   - cmake

